### PR TITLE
Add gRPC Kestrel configuration documentation

### PR DESCRIPTION
**Pull request by JamesNK**
Addresses https://github.com/grpc/grpc-dotnet/issues/476

Internal review URL: https://review.docs.microsoft.com/en-us/aspnet/core/grpc/aspnetcore?view=aspnetcore-3.0&branch=pr-en-us-14015&tabs=visual-studio

### CODE REVIEW

File: aspnetcore/grpc/aspnetcore.md [hunk 1]
```diff
@@ -47,16 +47,75 @@ gRPC requires the [Grpc.AspNetCore](https://www.nuget.org/packages/Grpc.AspNetCo
...
-[!code-csharp[](~/tutorials/grpc/grpc-start/sample/GrpcGreeter/Startup.cs?name=snippet&highlight=24)]
+[!code-csharp[](~/tutorials/grpc/grpc-start/sample/GrpcGreeter/Startup.cs?name=snippet&highlight=7,24)]
 
 ASP.NET Core middlewares and features share the routing pipeline, therefore an app can be configured to serve additional request handlers. The additional request handlers, such as MVC controllers, work in parallel with the configured gRPC services.
 
+### Configure Kestrel
+
+Kestrel gRPC endpoints:
+
+* Required HTTP/2.
```
Comment (Tratcher) [inline aspnetcore/grpc/aspnetcore.md]:
```suggestion
* Require HTTP/2.
```

File: aspnetcore/grpc/aspnetcore.md [hunk 2]
```diff
@@ -47,16 +47,75 @@ gRPC requires the [Grpc.AspNetCore](https://www.nuget.org/packages/Grpc.AspNetCo
...
+* Should be secured with HTTPS.
+
+#### HTTP/2
+
+Kestrel endpoints default to HTTP/1.1, and require additional configuration to enable HTTP/2. The *appsettings.json* defaults endpoints to the HTTP/2 connection protocol:
```
Comment (Tratcher) [inline aspnetcore/grpc/aspnetcore.md]: Kestrels 3.0 default is HTTP/1.1 and HTTP/2.0. That config disables HTTP/1.1.

Comment (JamesNK) [reply aspnetcore/grpc/aspnetcore.md]: This doc needs to be updated then: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel?view=aspnetcore-3.0&branch=pr-en-us-14015#http2-support

Comment (guardrex) [reply aspnetcore/grpc/aspnetcore.md]: I should be getting to the HTTP/2 update within a couple of weeks on https://github.com/aspnet/AspNetCore.Docs/issues/10141.

File: aspnetcore/grpc/aspnetcore.md [hunk 3]
```diff
@@ -47,16 +47,76 @@ gRPC requires the [Grpc.AspNetCore](https://www.nuget.org/packages/Grpc.AspNetCo
...
+* Each gRPC service is added to the routing pipeline through the `MapGrpcService` method.
 
-Each gRPC service is added to the routing pipeline through the `MapGrpcService` method:
-
-[!code-csharp[](~/tutorials/grpc/grpc-start/sample/GrpcGreeter/Startup.cs?name=snippet&highlight=24)]
+[!code-csharp[](~/tutorials/grpc/grpc-start/sample/GrpcGreeter/Startup.cs?name=snippet&highlight=7,24)]
 
 ASP.NET Core middlewares and features share the routing pipeline, therefore an app can be configured to serve additional request handlers. The additional request handlers, such as MVC controllers, work in parallel with the configured gRPC services.
 
+### Configure Kestrel
```
[the remark this answers is not in the document]
Comment (JamesNK) [reply aspnetcore/grpc/aspnetcore.md]: We can add it now. Do you have some text in mind?

**Comment by JamesNK** [comment]
@guardrex @scottaddie @Rick-Anderson Ready for language review and merge 🙏

**Comment by guardrex** [comment]
This targets `master`, but it looks like this should go to the staging branch `release/3.0-preview9`.

**Pull request merged**